templates/al2023/provisioners/install-efa.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o nounset
+set -o errexit
+
+if [ "$ACCELERATOR_VENDOR" != "neuron" ] && [ "$ACCELERATOR_VENDOR" != "nvidia" ]; then
+  exit 0
+fi
+
+##########################################################################################
+### Setup installer ######################################################################
+### https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-start.html#efa-start-enable ##
+##########################################################################################
+EFA_VERSION="latest"
+EFA_PACKAGE="aws-efa-installer-${EFA_VERSION}.tar.gz"
+EFA_DOMAIN="https://efa-installer.amazonaws.com"
+PARTITION=$(imds "/latest/meta-data/services/partition")
+
+if [ ${PARTITION} == "aws-iso" ]; then
+  EFA_DOMAIN="https://aws-efa-installer.s3.${AWS_REGION}.c2s.ic.gov"
+elif [ ${PARTITION} == "aws-iso-b" ]; then
+  EFA_DOMAIN="https://aws-efa-installer.s3.${AWS_REGION}.sc2s.sgov.gov"
+fi
+
+mkdir -p /tmp/efa-installer
+cd /tmp/efa-installer
+
+#https://github.com/amazonlinux/amazon-linux-2023/issues/243
+sudo dnf swap -y gnupg2-minimal gnupg2-full
+
+##########################################################################################
+### Download installer ###################################################################
+##########################################################################################
+curl -O ${EFA_DOMAIN}/${EFA_PACKAGE}
+
+curl -O ${EFA_DOMAIN}/aws-efa-installer.key && gpg --import aws-efa-installer.key
+curl -O ${EFA_DOMAIN}/${EFA_PACKAGE}.sig
+if ! gpg --verify ./aws-efa-installer-${EFA_VERSION}.tar.gz.sig &> /dev/null ;then
+    echo "EFA Installer signature failed verification!"
+    exit 2
+fi
+
+##########################################################################################
+### Install and cleanup ##################################################################
+##########################################################################################
+tar -xf ${EFA_PACKAGE} && cd aws-efa-installer
+sudo ./efa_installer.sh -y
```

Comment:
The user space libraries aren't required on the host:

```suggestion
sudo ./efa_installer.sh --minimal -y
```

Reply:
sounds good